{
  "uid" : "58fae4c285d578a3",
  "name" : "test_xfail",
  "fullName" : "tests.test_marks.TestMarks#test_xfail",
  "historyId" : "a44cab49732a0aa43a757be1adf99317",
  "time" : {
    "start" : 1591177964286,
    "stop" : 1591177964286,
    "duration" : 0
  },
  "description" : "\n        Indicate that you expect it to fail\n        这条用例执行失败时，测试结果被标记为xfail(expected to fail)，并且不打印错误信息。\n        这条用例执行成功时，测试结果被标记为xpassed(unexpectedly passing)\n        :return:\n        ",
  "descriptionHtml" : "<pre><code>    Indicate that you expect it to fail\n    这条用例执行失败时，测试结果被标记为xfail(expected to fail)，并且不打印错误信息。\n    这条用例执行成功时，测试结果被标记为xpassed(unexpectedly passing)\n    :return:\n</code></pre>\n",
  "status" : "skipped",
  "statusMessage" : "AssertionError: assert 0",
  "statusTrace" : "self = <tests.test_marks.TestMarks object at 0x10ea29910>\n\n    @pytest.mark.xfail\n    def test_xfail(self):\n        \"\"\"\n        Indicate that you expect it to fail\n        这条用例执行失败时，测试结果被标记为xfail(expected to fail)，并且不打印错误信息。\n        这条用例执行成功时，测试结果被标记为xpassed(unexpectedly passing)\n        :return:\n        \"\"\"\n>       assert 0\nE       assert 0\n\ntests/test_marks.py:40: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "tag",
    "value" : "xfail"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_marks"
  }, {
    "name" : "subSuite",
    "value" : "TestMarks"
  }, {
    "name" : "host",
    "value" : "LIUMENGJIAdeMacBook-Pro.local"
  }, {
    "name" : "thread",
    "value" : "78362-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_marks"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "a210e58f4277d8e9",
      "status" : "skipped",
      "statusDetails" : "AssertionError: assert 0",
      "time" : {
        "start" : 1591176793940,
        "stop" : 1591176793940,
        "duration" : 0
      }
    } ],
    "categories" : [ ],
    "tags" : [ "xfail" ]
  },
  "source" : "58fae4c285d578a3.json",
  "parameterValues" : [ ]
}